quantum/split_common/serial.h

```diff
@@ -1,26 +1,65 @@
-#ifndef MY_SERIAL_H
-#define MY_SERIAL_H
+#ifndef SOFT_SERIAL_H
+#define SOFT_SERIAL_H
 
 #include <stdbool.h>
 
 // /////////////////////////////////////////////////////////////////
 // Need Soft Serial defines in config.h
 // /////////////////////////////////////////////////////////////////
 // ex.
-//  /* Configuration of lower interface with the lower layer(hardware) of serial.c */
 //  #define SOFT_SERIAL_PIN ??   // ?? = D0,D1,D2,D3,E6
+//  OPTIONAL: #define SELECT_SOFT_SERIAL_SPEED ? // ? = 1,2,3,4,5
```

Comment:
Add a note here on mode 0?

Reply:
0 of SELECT_SOFT_SERIAL_SPEED is an experimental value. I experimented to confirm that there was enough room, but I think that I should not be used regularly. Therefore, it hid in the header file.